container/docker/handler.go

```diff
@@ -312,7 +314,7 @@ func (self *dockerContainerHandler) ListContainers(listType container.ListType)
 		}
 
 		ref := info.ContainerReference{
-			Name:      FullContainerName(c.ID),
+			Name:      FullContainerName(c.ID, self.cgroupParent),
```

Comment:
Is this not setting the cgroup parent the same for all containers? Since this for is looping over all docker containers (self.client is a reference to the Docker client), but self.cgroupParent is only for the current container.